storage/influxdb/influxdb.go

```diff
@@ -299,14 +319,26 @@ func newStorage(
 		return nil, err
 	}
 
+	//Reuse cadvisor flag for whitelisted container labels
+	var argWhitelistedLabels = flag.Lookup("whitelisted_container_labels").Value.(flag.Getter).Get().(string)
```

Comment:
I would rather explicitly plumb this through, and do all of the flag parsing when cAdvisor starts if possible.

Reply:
It can be done, but I'm not so familiar with the patterns and preferences here yet.

Are you suggesting to pass it down on the `storage` constructor or maybe add it to the `StorageDriver.AddStats` interface? 

I was trying to avoid modifying anything on other storage drivers as I have no context on then but it makes sense that they also respect the labels when storing if possible.